### DISCUSSION TITLE
In mitmweb, how to distinguish requests from multiple devices

### DISCUSSION THREAD

**Discussion by Pactortester** [mitmproxy/mitmproxy#5251] [Q&A] [answered]
In mitmweb, how to distinguish requests from multiple devices？

<img width="1324" alt="image" src="https://user-images.githubusercontent.com/29191106/162227602-eea32ede-f483-4795-8a91-96744473258b.png">

**Comment by mhils** [comment, marked as answer]
If you click on "Connection" on the right, you should be able to see the source IP address. You can also use filters to only display selected clients or highlight them.

**Reply by Pactortester** [reply]
thanks!
